src/index.html

```diff
@@ -6,9 +6,58 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+
+      <header class="header">
+        <div class="header__logo">
+          <img class="header__logo-img"
+          src="images/logo.png"
+          alt="logo"
+          >
+        </div>
+        <nav class="header__nav">
+          <ul class="header__nav-list">
+            <li class="header__nav-item">
+              <a class="header__nav-link is-active"
+              href="#"
+              >Apple</a>
+            </li>
+            <li class="header__nav-item">
+              <a class="header__nav-link" href="#">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="header__nav-item">
              <a class="header__nav-link is-active"
              href="#"
              >Apple</a>
            </li>

            <li class="header__nav-item">
              <a class="header__nav-link" href="#">Samsung</a>
            </li>
```

Add a blank line between multiple sibling blocks

Fix everywhere